Library/.rubocop.yml

```diff
@@ -5,7 +5,9 @@ AllCops:
     - '**/vendor/**/*'
   DisplayCopNames: false
 
-require: ./Homebrew/rubocops.rb
+require: 
+  - rubocop-rspec
```

Comment:
I think this should be in `Library/Homebrew/.rubocop.yml` instead as it's not needed for formulae and is only currently only used by the `.rubocop_todo.yml`. Thoughts? Also, what does this line output when you don't have `rubocop-rspec` installed?

Reply:
I agree, I'll move it a recommit. The error that gets spit out is
```
➜  Homebrew git:(moverubocopoption) rubocop
cannot load such file -- rubocop-rspec
/usr/local/Cellar/ruby/2.5.1/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/usr/local/Cellar/ruby/2.5.1/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_loader_resolver.rb:15:in `block in resolve_requires'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_loader_resolver.rb:11:in `each'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_loader_resolver.rb:11:in `resolve_requires'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_loader.rb:43:in `load_file'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_loader_resolver.rb:139:in `block in base_configs'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_loader_resolver.rb:138:in `map'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_loader_resolver.rb:138:in `base_configs'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_loader_resolver.rb:22:in `resolve_inheritance'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_loader.rb:49:in `load_file'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_loader.rb:82:in `configuration_from_file'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/config_store.rb:44:in `for'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/cli.rb:187:in `apply_default_formatter'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/lib/rubocop/cli.rb:40:in `run'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/exe/rubocop:13:in `block in <top (required)>'
/usr/local/Cellar/ruby/2.5.1/lib/ruby/2.5.0/benchmark.rb:308:in `realtime'
/usr/local/lib/ruby/gems/2.5.0/gems/rubocop-0.59.2/exe/rubocop:12:in `<top (required)>'
/usr/local/bin/rubocop:23:in `load'
/usr/local/bin/rubocop:23:in `<main>'
➜  Homebrew git:(moverubocopoption)
```
a bit silly but I would assume this would be a rubocop enhancement failing more nicely when trying to load non-exisiting extensions.